### ISSUE TITLE
1.1.7 with GUI - GUI crashing after startup

### ISSUE THREAD

**Issue by Eruthon** [rustdesk/rustdesk-server#192]
**Describe the bug**
I tried to install the new 1.1.7 version with GUI. After installation it just blink its window for a split second, leaving an icon in taskbar, which hides when I hover over it. No logs are generated.
I tried to remove old RustDesk folder from Program Files, removing old services via nssm and creating new ones with the provided nssm run.cmd file. I then restored my old keys to bin folder and I can connect to my clients again without issue. Only the GUI doesn't work for some reason.

**Describe the environment**
 - Install environment: Windows Server 2022 Standard
 - Latest (14. Feb) RustDesk Server 1.1.7 with GUI 

**How to Reproduce the bug**
1. Download new Windows installer
2. Extract files
3. Install via setup .exe file
4. Start server
5. No GUI can be seen

**Expected behavior**
GUI should remain open after starting it.

**Additional context**
- 
![MobaXterm_NJNUGNPsoS](https://user-images.githubusercontent.com/37985268/218998846-79168f09-42cf-4ae1-8807-bc0284dda6b3.gif)

**Notes**
- (EDIT#1) the uninstall executable does not remove any files from installation folder

**Comment by rustdesk** [comment]
@elilchen

**Comment by elilchen** [comment]
@Eruthon Thank for you gif! That helps a lot.

**Comment by rustdesk** [comment]
Will release soon

**Comment by rustdesk** [comment]
Try out new build. https://github.com/rustdesk/rustdesk-server/releases/download/1.1.7-1/rustdesk-server-windows-x86_64.zip

**Comment by Eruthon** [comment]
Just commenting on reinstalling of the new version - had to restart the windows server: the installer did not stop the services, so it couldn't install new hbbr and hbbs executabes; even after stopping the services, the hbbr process still remained running (visible in the details tab in task manager) and I couldn't stop the process by ending task. Set the services to not start autoamtically, will report with new info after restart.

**Comment by Eruthon** [comment]
So after the restart the hbbr still runs even while being set to manual service start...? I don't get it. I'll just delete the old hbbr.exe on system restart and let you know.

**Comment by Eruthon** [comment]
Ok, after the file deletion the setup finished corectly and the GUI showed up. After that I started the services, but nothing were showing in the log window. I clicked on Service>Start and after that the cmd showed some errors regarding not being able to start services, but the cmd disappeared. But the window started to show logs about private keys being loaded, some rendezvous server stuff etc.
Seems both the hbbs.log and hbbr.log are showing correctly now. No logs in .err logs for now.
